src/systemd/tftpd@.service

```diff
@@ -7,7 +7,7 @@ RequiresMountsFor=/run
 Type=forking
 #NotifyAccess=main
 EnvironmentFile=-/etc/default/tftpd%I
-ExecStart=/usr/sbin/in.tftpd "$DAEMON_ARGS"
+ExecStart=/bin/sh -c "${VRF_ARGS} /usr/sbin/in.tftpd ${DAEMON_ARGS}"
```

Comment:
Is there a particular reason this uses `/bin/sh` as executor and not like the other patterns? Just trying to understand.

* https://github.com/vyos/vyos-1x/blob/current/data/templates/ssh/override.conf.tmpl#L9
* https://github.com/vyos/vyos-1x/blob/current/data/templates/ntp/override.conf.tmpl#L9

Reply:
Yes, we can not use ENV arg as the first param if it is not defined exactly in service. In this case, it is not possible to define VRF_ARGS here and we are defining it in /etc/default/tftpN files.